src/Data/NonEmpty.purs

```diff
@@ -54,7 +60,7 @@ oneOf :: forall f a. (Alternative f) => NonEmpty f a -> f a
 oneOf (NonEmpty a fa) = pure a <|> fa
 
 instance showNonEmpty :: (Show a, Show (f a)) => Show (NonEmpty f a) where
-  show (NonEmpty a fa) = "(NonEmpty " ++ show a ++ " " ++ show fa ++ ")"
+  show (NonEmpty a fa) = "NonEmpty (" ++ show a ++ ") (" ++ show fa ++ ")"
```

Comment:
Yeah, I think we have an issue somewhere discussing ideas for it - without default typeclass members it would kinda be a pain at the moment. Although I guess we could do the provide-default-function approach. We perhaps should have broken that in 0.7... although if we do have default class members at some point, we can do it without breaking everything I guess.